src/main/java/com/hubspot/maven/plugins/dependency/management/DependencyManagementRule.java

```diff
@@ -9,6 +9,7 @@
 public class DependencyManagementRule implements EnforcerRule {
 
   private RequireManagement requireManagement = new RequireManagement();
+  private boolean fail;
```

Comment:
How does this get set?

Reply:
Plugin magic?

Reply:
For the enforcer plugin, the fail flag gets set at a higher level. I'm not sure if we have a way to access it inside our rule. I'd say to probably just log stuff as a warning for the enforcer rule.